source/plugin.js

```diff
@@ -0,0 +1,27 @@
+module.exports = [
+  {
+    name: 'clean-project-auto',
```

Comment:
The explanation for the why of these two options is necessary in the README.

Reply:
Totally agreed - I just wanted to see if you receptive before I did the administrivia

Reply:
please add a `description` to the tasks for more helpful `--help` output of the CLI

Reply:
fantastic - I'll get to it, thanks!